docs/get-started/on-device-training.md

```diff
@@ -0,0 +1,49 @@
+---
+title: On-Device Training
+parent: Get Started
+nav_order: 12
+---
+
+# On-Device Training with ONNX Runtime
+{: .no_toc }
+`On-Device Training` refers to the process of training a model on an edge device, such as mobile phones, embedded devices, gaming consoles, web browsers, etc. This is in contrast to training a model on a server or a cloud. Training on the device
+
+- Can improve data privacy and security, especially when working with sensitive data that cannot be shared with a server or a cloud.
+- It is also useful for the task of personalization where the model needs to be trained on the user's data.
```

Comment:
Should be "Is useful..." Also the ordering should be personalization first and privacy second.